File: CyFile-Android/app/src/test/java/at/tugraz/tc/cyfile/domain/NoteUnitTest.java
package at.tugraz.tc.cyfile.domain;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;
import nl.jqno.equalsverifier.Warning;

public class NoteUnitTest {


    @Test
    public void testEqualVerifier() {
        EqualsVerifier.forClass(Note.class)
                .suppress(Warning.ALL_FIELDS_SHOULD_BE_USED)
                .suppress(Warning.NONFINAL_FIELDS)
                .verify();
    }
}

File: CyFile-Android/app/src/test/java/at/tugraz/tc/cyfile/domain/NoteUnitTest.kt
package at.tugraz.tc.cyfile.domain

import at.tugraz.tc.cyfile.BaseUnitTest
import nl.jqno.equalsverifier.EqualsVerifier
import nl.jqno.equalsverifier.Warning
import org.junit.Test

class NoteUnitTest : BaseUnitTest() {


    @Test
    fun testEqualVerifier() {
        EqualsVerifier.forClass(Note::class.java)
                .suppress(Warning.ALL_FIELDS_SHOULD_BE_USED)
                .suppress(Warning.NONFINAL_FIELDS)
                .verify()
    }
}
